modules/plugin/jdbc/jdbc-postgis/src/test/java/org/geotools/data/postgis/PostgisPrimaryKeyFinderOnlineTest.java

```diff
@@ -18,4 +21,36 @@ protected JDBCPrimaryKeyFinderTestSetup createTestSetup() {
         params.put(JDBCDataStoreFactory.PK_METADATA_TABLE.key, "gt_pk_metadata");
         return params;
     }
+
+    public void testPrimaryKeyMetadataView() throws Exception {
+        PostgisPrimaryKeyFinderTestSetupWrapper testSetup =
```

Comment:
This is improper usage of the testing framework, the test setup is called once at the beginning of the test, period.

Reply:
Done.